ssrando.py

```diff
@@ -120,6 +120,15 @@ def _get_rando_hash(self):
         current_hash.update(str(self.seed).encode("ASCII"))
         current_hash.update(self.options.get_permalink().encode("ASCII"))
         current_hash.update(VERSION.encode("ASCII"))
+        if self.options["hint-distribution"] == "Custom":
+            if not os.path.exists("custom_hint_distribution.json"):
+                raise Exception(
+                    "Custom hint distribution file not found. Make sure custom_hint_distribution.json exists at the same location as the randomizer"
+                )
+            with open("custom_hint_distribution.json") as f:
+                lines = f.readlines()
+                for line in lines:
+                    current_hash.update(line.strip(" \n").encode("ASCII"))
```

Comment:
```suggestion
            with open("custom_hint_distribution.json") as f:
                normalized_json = json.dumps(json.load(f))
                current_hash.update(normalized_json.encode('ASCII'))
```
Checking a bit for how to hash dictionaries in python, it seems like most recomment just dumping to json and hashing that. This should also get rid of line ending problems